test/parallel/test-fork-send-returns-boolean.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+
+const n = fork(common.fixturesDir + '/empty.js');
+
+const rv = n.send({ hello: 'world' });
+assert.strictEqual(rv, true);
```

Comment:
On the `-e` thing, FWIW, it hangs for me using Node 4.2.1, 0.12.7, 0.10.40, and 0.8.28. So definitely not a *recent* regression.